tests/puppeteer/pages/FO/FObasePage.js

```diff
@@ -133,4 +134,12 @@ module.exports = class Home extends CommonPage {
   async getFooterLinksBlockTitle(position) {
     return this.getTextContent(this.wrapperTitle.replace('%POSITION', position));
   }
+
+  /**
+   * Get Page content
+   * @return {Promise<textContent>}
+   */
+  getPageContent() {
```

Comment:
I guess this is the same kind of method than in `BoBasePage` ?

Reply:
Yes it's the same method in `commonPage` -> `getTextContent`
I will chage it :)

Reply:
Maybe we can move it to `BasePage` ? (Or `CommonPage` I don't remember the parent)